tensorflow/contrib/lite/kernels/gather_test.cc

```diff
@@ -76,6 +76,27 @@ TEST(GatherOpTest, Shuffle) {
               ElementsAreArray(ArrayFloatNear({0.7, 0.8, -2, 0.2})));
 }
 
+TEST(GatherOpTest, Index) {
```

Comment:
Does `Index` apply it's 0D? Should we call this test case something like `Test0DIndex`?

Reply:
Yes, you are right. It will be more straightforward.

Reply:
Thanks. Could you update the PR?